Resources/Prototypes/_Sunrise/Entities/Objects/Specific/Robotics/borg_tools.yml

```
@@ -167,20 +167,59 @@
     generated:
       reagents:
       - ReagentId: DexalinPlus
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)